src/mapgen_functions.cpp

```diff
@@ -152,7 +152,7 @@ void init_mapgen_builtin_functions() {
     mapgen_cfunction_map["basement_spiders"] = &mapgen_basement_spiders;
     mapgen_cfunction_map["office_doctor"] = &mapgen_office_doctor;
     mapgen_cfunction_map["sub_station"] = &mapgen_sub_station;
-    mapgen_cfunction_map["s_garage"] = &mapgen_s_garage;
+//   mapgen_cfunction_map["s_garage"] = &mapgen_s_garage;
```

Comment:
Don't bother commenting stuff out, if it's unneeded it's just plain unneeded. We can roll back the history if we really need to, no need to keep stuff 'alive but commented'.